src/components/header/JourneySection.tsx

```diff
@@ -29,11 +29,11 @@ const JourneySection = () => {
 
           <div className="m-2 p-3 bg-gray-400 text-white flex rounded-lg justify-around text-xl  items-center">
             {steps.map((step, index) => (
-              <div key={index}>
```

Comment:
Can you sort the steps so that the arrangement match the one on design as it is steps their arrangement highly matter and remove the available typos

Reply:
i sorted them, but becouse they are shuffled they can not be sorted 100% we can not match the order unless we hardcode them